src/contrib/cluster/Akka.Cluster.Sharding/ClusterSharding.cs

```diff
@@ -277,7 +277,8 @@ public ClusterSharding(ExtendedActorSystem system)
         /// <returns>TBD</returns>
         public static Config DefaultConfig()
         {
-            return ConfigurationFactory.FromResource<ClusterSharding>("Akka.Cluster.Sharding.reference.conf");
+            return ConfigurationFactory.FromResource<ClusterSharding>("Akka.Cluster.Sharding.reference.conf")
+                .WithFallback(DistributedData.DistributedData.DefaultConfig());
```

Comment:
Changed how configuration works. Instead of copying the structure of `akka.harding.distributed-data` into sharding config,`akka.cluster.sharding.distributed-data` works as an override for `akka.cluster.istributed-data` instead.
This is done to minimize problems where we have 2 different distributed data settings in the system, potentially bugging the system.